app/assets/javascripts/src/Metamaps.js.erb

```diff
@@ -14,6 +14,7 @@ Metamaps.VERSION = '<%= METAMAPS_VERSION %>'
 /* erb variables from rails */
 Metamaps.Erb = {}
 Metamaps.Erb['REALTIME_SERVER'] = '<%= ENV['REALTIME_SERVER'] %>'
+Metamaps.Erb['RAILS_ENV'] = '<%= ENV['RAILS_ENV'] %>'
```

Comment:
We'll have to be careful with this variable, because anyone could just run `Metamaps.Erb.RAILS_ENV = 'development'` and then they will get different behaviour on metamaps.cc. If we add more complicated stuff based on this we'll have to keep it in mind